perllib/FixMyStreet/Cobrand/Bexley/Waste.pm

```diff
@@ -1366,22 +1485,14 @@ sub waste_request_form_update_field_list {
     return $fields;
 }
 
-sub waste_request_form_first_next {
-    my $self = shift;
-
-    # Above-shop properties get a single default reason, so no need to send
-    # them to reason selection
-    return $self->{c}->stash->{property}{above_shop}
-        ? 'about_you'
-        : 'request_reason';
-}
-
 sub waste_munge_request_form_pages {
```

Comment:
Is this function needed any more? You could set first_page depending upon household_size_check in your main `construct_bin_request_form` above.